src/Symfony/Component/HttpKernel/Controller/ContainerControllerResolver.php

```diff
@@ -23,6 +23,9 @@
  */
 class ContainerControllerResolver extends ControllerResolver
 {
+    /**
+     * @var ContainerInterface
+     */
```

Comment:
The shouldn't be needed, as it's type hinted in the constructor

Reply:
Hi, i  so i can suggest `protected ContainerInterface $container;`

Reply:
Changing a protected property to be typed is a BC break (for child classes the redeclare it)